aten/src/ATen/native/ComplexHelper.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <ATen/ATen.h>
+
+namespace at { namespace native {
+
+inline std::vector<int64_t> computeStrideForComplex(IntArrayRef oldstride) {
```

Comment:
why do we need inline here? I would rather put definitions into ComplexHelper.cpp and keep only declarations here as usual.

Reply:
I get the multiple definition error otherwise: https://github.com/pytorch/pytorch/issues/20686#issuecomment-593002293